process/slack_etiquette.md

```diff
@@ -0,0 +1,64 @@
+# **`Draft language. Needs Editing.`**
+
+This was copied (_with permission_) from the Codebuddies (www.codebuddies.org) community.  
+While some of these are **great** for PyLadies, some of these points need editing for our commuity.
+
+Please propose or edit as you see fit.
+
+______________________________________
+______________________________________
+
+## The following guidelines can help you get the most from our Slack community.
+
+.
+### :star2: Every member of CodeBuddies is a _*person*_ 
+ _This is a safe space. No question is "stupid.", no topic too "basic"._  
+.
+This community is intended for self-directed learners, CS students, programming enthusiasts, and professional developers to share knowledge and techniques. The admins, moderators, and everyone else in CodeBuddies are **donating their spare time; nobody’s being paid.**  Our suggestions/answers may not always be correct, & you may not always get an answer. Treat us like somebody friendly you don’t know who’s standing next to you offering help or suggestions.
```

Comment:
*Edit*: The language below I think is a great way to frame how to participate. I do think per @Mariatta and @BethanyG concerns that maybe we can simply say "this PyLadies Slack is governed by the PSF Code of Conduct." and use this language to help set expectations:

```
As a fiscal sponsoree of the Python Software Foundation, all PyLadies spaces are governed by the PSF Code of Conduct. You can read the Code of Conduct [here](https://www.python.org/psf/conduct/).

This community is intended first and foremost any self-identifying PyLadies member. While allies and friends of PyLadies may participate in the PyLadies Slack the PyLadies Slack is intended to be a community built for and by self-identifying PyLadies. 

Additionally, the admins, moderators, and everyone else in PyLadies are **donating their spare time; nobody’s being paid.**  Our suggestions/answers may not always be correct, & you may not always get an answer. Treat us like somebody friendly you don’t know who’s standing next to you offering help or suggestions.
```

Reply:
I think that's excellent language!  Updated.